### ISSUE TITLE
[BUG] Removing Game-Specific AMXX Breaks base AMXX

### ISSUE THREAD

**Issue by fysiks1** [GameServerManagers/LinuxGSM#3251]
## User Story

As an AMX Mod X user, I want to remove a the game-specific AMX Mod X files so that the base AMX Mod X still works.

## Basic info

* **Distro:** Ubuntu 20.04.2 LTS
* **Game:** Day of Defeat
* **Command:** mods-remove
* **LinuxGSM version:** 21.1.3

## Further Information

When you remove the game-specific files for AMX Mod X, it would seem logical that I would then have a working version of the base version of AMX Mod X.  However, some base AMX Mod X files are overwritten by custom versions that come with the game-specific files of AMX Mod X.

## To Reproduce

Steps to reproduce the behaviour:
1. Install Day of Defeat
2. Install MetaMod (metamod)
3. Install AMX Mod X (amxmodx)
4. Install AMX Mod X for DOD (amxmodxdod)
5. Run server and run "meta list" command in console to verify all modules loaded
6. Stop server
7. Remove amxmodxdod
8. Run server and run "meta list" command in console and see that AMX Mod X fails to load.

## Expected behaviour
It seems logical that it would restore the original files from amxmodx that were overwritten and subsequently removed by installing and removing amxmodxdod.

**Comment by fysiks1** [comment]
Probably not really all that important since you probably won't need to actually run a plain base version when you're running on a game that has specific files.

**Comment by tkrn** [comment]
You're right. It's a good idea to re-write using the amxmodx base after the removal of a game-specific amxmodx. It's not that much effort to have it do a manual overwrite. The use case is slim and likely unknown but it's the right thing to do. I will implement it at some point but it'll be a low priority.
